src/datepicker/datepicker.stories.ts

```diff
@@ -7,13 +7,86 @@ import {
 	text,
 	boolean
 } from "@storybook/addon-knobs/angular";
+import {
+	FormBuilder,
+	FormGroup,
+	Validators,
+	FormsModule,
+	ReactiveFormsModule
+} from "@angular/forms";
+import {
+	Component,
+	Input,
+	Output,
+	EventEmitter
+} from "@angular/core";
 import { DatePickerModule } from "../";
 
+
+@Component({
+	selector: "app-date-picker",
+	template: `
+		<form [formGroup]="rForm">
+			<ibm-date-picker
+				formControlName="single"
+				label="Date Picker Label"
+				invalidText="Invalid date format"
+				[invalid]="invalidSingle"
+				(valueChange)="valueChange.emit($event)">
+			</ibm-date-picker><br>
+			<ibm-date-picker
+				range="true"
+				formControlName="range"
+				label="Date Picker Label"
+				rangeLabel="Date Picker Label"
+				invalidText="Invalid date format"
+				[pattern]="pattern"
+				[invalid]="invalidRange"
+				(valueChange)="valueChange.emit($event)">
+			</ibm-date-picker>
+		</form>
+	`
+})
+class DatePickerStory {
+	@Output() valueChange = new EventEmitter();
+
+	get invalidSingle() {
+		return this.rForm.controls["single"].invalid && this.rForm.controls["single"].touched;
+	}
+
+	get invalidRange() {
+		return this.rForm.controls["range"].invalid && this.rForm.controls["range"].touched;
+	}
+
+	rForm: FormGroup;
+
+	constructor(protected formBuilder: FormBuilder) {
+		this.rForm = this.formBuilder.group({
+			single: [
+				[(new Date().getMonth() + 1) + "/" + new Date().getDate() + "/" + new Date().getFullYear()],
```

Comment:
Couldn't this just be `new Date()`?

Reply:
`new Date()` will also get the current time, if we want the start of the day, then no?